aws-iot-device-sdk-java/pom.xml

```diff
@@ -38,7 +38,7 @@
     <dependency>
       <groupId>org.eclipse.paho</groupId>
       <artifactId>org.eclipse.paho.client.mqttv3</artifactId>
-      <version>[1.2.2]</version>
+      <version>[1.2.5]</version>
```

Comment:
Unless there's a specific other fix or change we want in 1.2.5, should we consider only updating to 1.2.4? It looks as though 1.2.5 introduced MQTT5 and that may add things we don't want to in the V1 Client. Might be worth considering but also might not be an issue at all.

Reply:
Good catch - I totally missed that. I looked at the fixes for 1.2.5 in the change log and it is almost all MQTT5 related. I'll bump it down to 1.2.4, so we get all the latest fixes but without the MQTT5 stuff we do not need.
Strange that a library called client.mqttv3 would include MQTT5 on a minor version bump... 🤷